queries.sql

```diff
@@ -8,3 +9,55 @@ SELECT name , escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/*Day Two */
+
+/* Transaction - BEGIN, UPDATE and COMMIT */
+
+/*UPDATE*/
+BEGIN TRANSACTION;
+UPDATE animals SET species='unspecified';
+
+SELECT species FROM animals;
+ROLLBACK TRANSACTION;
+SELECT species FROM animals;
+
+UPDATE animals SET species='digimon'
+	WHERE name::text LIKE '%mon';
+	
+UPDATE animals SET species='pokemon'
+	WHERE species ISNULL;
+	
+COMMIT TRANSACTION;
+
+SELECT * FROM animals ;
+
+/*Delete and Savepoint*/
+
+BEGIN TRANSACTION;
+
+SAVEPOINT SP1;
+ 
+ DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT SP2;
+
+UPDATE animals SET weight_kg = weight_kg * -1;
+
+ROLLBACK TRANSACTION TO SP1;
+
+UPDATE animals set weight_kg = weight_kg * -1 
+	WHERE weight_kg < 0;
+	
+COMMIT TRANSACTION;
+ 
+SELECT * FROM animals 
```

Comment:
- [ ] You seem to be missing a transaction for one of the questions that reads as "Inside a transaction delete all records in the animals table, then roll back the transaction." so please add that along with the screenshot.
- [ ] For the highlighted code above you should roll back the transaction to the saved point after you perform the delete operation which is at "SP2" so pelase update this as well.